server/services/search.js

```diff
@@ -2,10 +2,10 @@ const utils = require('../utils');
 
 const createSearchService = ({
   searchRepository,
-  getEstablishmentFormattedName = utils.getEstablishmentFormattedName,
+  getEstablishmentSearchName = utils.getEstablishmentSearchName,
 }) => {
   function find({ query, limit, from, establishmentId }) {
-    const prison = getEstablishmentFormattedName(establishmentId);
+    const prison = getEstablishmentSearchName(establishmentId);
```

Comment:
Do we set `prison` first to highlight errors? Otherwise, it should probably go below the `query === ''` check following the whole _return early_ ideology.

Reply:
meh we don't error on it so no. 

I agree in principal but this is big enough already